src/bindings/python/tests/test_runtime/test_core.py

```diff
@@ -429,3 +429,72 @@ def test_infer_new_request_return_type(device):
     assert arr.shape == (10,)
     assert arr.dtype == "float32"
     assert arr.nbytes == 40
+
+
+def test_read_model_node_names_with_spaces():
```

Comment:
@artkuli Why did you add a python test?
I think it should be a C++ test.

Moreover can we add an additional test when we add output in the runtime?

Reply:
@ilyachur The problem was related to the reading parameters from IR not directly to adding outputs in the runtime. The `std::stringstream` inside template method `str_to_container` cut the spaces in names so I wrote specifier method with different behavior when we would like to cast string to the `std::vector<std::string>`. I added Python test because in my opinion, in this case the bigger overview is better but obviously I can add also the test on C++ side 😊

Reply:
@ilyachur  C++ tests added